datafusion/expr/src/logical_plan/builder.rs

```diff
@@ -1592,7 +1592,47 @@ impl TableSource for LogicalTableSource {
 
 /// Create a [`LogicalPlan::Unnest`] plan
 pub fn unnest(input: LogicalPlan, columns: Vec<Column>) -> Result<LogicalPlan> {
-    unnest_with_options(input, columns, UnnestOptions::new())
+    unnest_with_options(input, columns, UnnestOptions::default())
+}
+
+pub fn get_unnested_columns(
```

Comment:
Could you please add some documentation explaining what this function does?

Also, I wonder if it needs to be `pub` or if we could leave it `pub(crate)` or not pub?